airbyte-workers/src/main/java/io/airbyte/workers/process/AsyncOrchestratorPodProcess.java

```diff
@@ -71,6 +72,7 @@ public AsyncOrchestratorPodProcess(
                                      final String secretName,
                                      final String secretMountPath,
                                      final String containerOrchestratorImage,
+                                     final String containerOrchestratorImagePullPolicy,
```

Comment:
let's move this to the `KubePodInfo` object on line 69. This helps us keep all the kube related info in that object. Arguably, we should also move the `containerOrchestratorImage` information into that object, so the inputs into this function are around configuring the orchestrator pod's state location rather than the underlying Kubernetes layer.

Reply:
Created a `KubeContainerInfo` record to hold container related information and a field in the `KubePodInfo` for the main container info.